### PR TITLE
Adding new chat manage delete permissions

### PR DESCRIPTION
**Pull request by sweta-thapliyal**

### CODE REVIEW

**Review by AkJo — APPROVED** [review]

**Review by AkJo — APPROVED** [review]

File: changelog/Microsoft.Teams.GraphSvc.json [hunk 1]
```diff
@@ -202,6 +202,42 @@
...
+      "SubArea": ""
+    },
+    {
+      "ChangeList": [
+        {
+          "Id": "05DF0873-86C7-4B55-99AB-35AE3A4AFB09",
+          "ApiChange": "Resource",
+          "ChangedApiName": "Chat.ManageDeletion.All",
+          "ChangeType": "Addition",
+          "Description": "Added new permissions [Chat.ManageDeletion.All](https://docs.microsoft.com/en-us/graph/permissions-reference#Chat-permissions).",
```
Comment (isvargasmsft) [inline changelog/Microsoft.Teams.GraphSvc.json:230]: Are these permissions used by an API set? We usually add the changelog entries about permissions referencing the method we are adding the permission to. For an example, see example 14 here: https://dev.azure.com/msazure/One/_wiki/wikis/Microsoft%20Graph%20Partners/211742/Changelog?anchor=changelist-record-examples

File: concepts/permissions-reference.md [hunk 2]
```diff
@@ -547,6 +547,7 @@ For more complex scenarios involving multiple permissions, see [Permission scena
 |_Chat.Read_ |Read your chat messages.  |Allows an app to read your 1:1 or group chat messages in Microsoft Teams, on your behalf. |No | No |
 |_Chat.ReadBasic_ |Read names and members of user chat threads.  |Allows an app to read the members and descriptions of 1:1 and group chats threads, on behalf of the signed-in user. |No | No |
 |_Chat.ReadWrite_ |Read your chat messages and send new ones.  |Allows an app to read and send your 1:1 or group chat messages in Microsoft Teams, on your behalf. |No | No |
+|_Chat.ManageDeletion.All_ |Delete and recover deleted chats.  |Allows the app to delete and recover deleted chats, on behalf of the signed-in user. |Yes | No |
```
Comment (isvargasmsft) [inline concepts/permissions-reference.md:550]: Are these permissions used by an API set? Did we update those permissions tables already? If not, please add the new permissions to the method topics that use them.

**Review by isvargasmsft — CHANGES_REQUESTED** [review]
Hello @sweta-thapliyal, 

Thank you very much for these updates. 

Please clarify if there are methods using these properties already. 

Thank you!

**Review by isvargasmsft — APPROVED** [review]
Methods will be introduced to Microsoft Graph eventually.

**Pull request merged**